reference/entry.json

```diff
@@ -2414,6 +2414,44 @@
         ]
       }
     },
+    "/organizers/{organizerId}": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/organizerId"
+        }
+      ],
+      "delete": {
+        "summary": "Delete organizer",
+        "description": "Delete the organizer with the given `organizerId`.",
```

Comment:
Nit: This operation requires a special permission `ORGANISATIES_BEHEREN`. The creator of the organizer cannot delete it themself. So maybe good to document that here, even if this is marked as internal. (That way our frontend devs also know who can use this and who not.) We can probably also improve the same description for updating facilities on events/places now that I think about it 😄

Reply:
That was even added to the description of the ticket but I missed it 😅

Reply:
I added a info note.